docs/pom.xml

```diff
@@ -39,8 +39,8 @@
         <vale.image>docker.io/jdkato/vale:v2.15.5</vale.image>
 
         <!-- Keep these in sync with the full versions in bom/application/pom.xml -->
-        <hibernate-orm.version-for-documentation>6.4</hibernate-orm.version-for-documentation>
-        <hibernate-search.version-for-documentation>7.0</hibernate-search.version-for-documentation>
+        <hibernate-orm.version-for-documentation>6.5</hibernate-orm.version-for-documentation>
+        <hibernate-search.version-for-documentation>7.1</hibernate-search.version-for-documentation>
```

Comment:
Couldn't we use the Maven plugin we use to split the versions?

Reply:
BTW, I don't mind keeping it that way.

Reply:
I'll look into this, but in a separate PR.

Reply:
https://github.com/quarkusio/quarkus/pull/40964